cpp/doxygen/main_page.md

```diff
@@ -28,7 +29,7 @@ to include KvikIO in your CMake projects.
 
 ### Conda/Mamba
 
-We strongly recommend using `mamba <https://github.com/mamba-org/mamba>`_ inplace of conda, which we will do throughout the documentation.
+We strongly recommend using [mamba](https://github.com/mamba-org/mamba) inplace of conda, which we will do throughout the documentation.
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏

```suggestion
We strongly recommend using [mamba](https://github.com/mamba-org/mamba) in place of conda, which we will do throughout the documentation.
```